interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.scss

```diff
@@ -0,0 +1,3 @@
+.active {
+    background-color: rgb(244, 231, 254);
```

Comment:
Please only use the colours from [variables.scss](https://github.com/rodekruis/IBF-system/blob/master/interfaces/IBF-dashboard/src/theme/variables.scss). [Refer to the Ionic documentation on how to use colours.](https://ionicframework.com/docs/theming/colors)

Reply:
Taken care